app/ui/src/app/integration/list/list.component.html

```diff
@@ -34,7 +34,7 @@
           <span class="fa fa-angle-right"></span>
           <span class="icon">
             <img class="syn-icon-small" src="./../../assets/icons/multi-flow.connection.png" aria-hidden="true">
-            <span class="badge">{{ integration.flows.length }}</span>
+            <span class="syn-badge" tooltip="{{ integration.flows.length }} implemented flows" container="body" placement="right">{{ integration.flows.length }}</span>
```

Comment:
`{{ integration.flows.length }}` will yield the number of flows, aka operations, coming from the spec. AFAIK there is no clear way to know the number of _implemented_ flows. @nicolaferraro is there any property we can use to figure out if an operation is implemented or not, but checking if the last step returns anything different from a 501?

Reply:
We have talked about that in a call. I remember that 501 should be the "flag" that distinguish a implemented operation from a non-implemented one.